[2022-03-18 04:46:41,174] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2010-03-07T04:00:00+00:00 [None]> on host a2ba0547c319
[2022-03-18 04:46:41,390] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2010-03-07T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2010-03-07T04:00:00+00:00
[2022-03-18 04:46:45,313] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2010-03-07T04:00:00+00:00 [queued]>
[2022-03-18 04:46:45,328] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2010-03-07T04:00:00+00:00 [queued]>
[2022-03-18 04:46:45,330] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:45,331] {taskinstance.py:1239} INFO - Starting attempt 5 of 5
[2022-03-18 04:46:45,332] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:45,361] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2010-03-07 04:00:00+00:00
[2022-03-18 04:46:45,372] {standard_task_runner.py:52} INFO - Started process 644 to run task
[2022-03-18 04:46:45,379] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2010-03-07T04:00:00+00:00', '--job-id', '6309', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmp1rcjbhgs', '--error-file', '/tmp/tmpvb4tdaze']
[2022-03-18 04:46:45,381] {standard_task_runner.py:77} INFO - Job 6309: Subtask get_pbp_data
[2022-03-18 04:46:45,535] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2010-03-07T04:00:00+00:00 [running]> on host a2ba0547c319
[2022-03-18 04:46:45,704] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2010-03-07T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2010-03-07T04:00:00+00:00
[2022-03-18 04:46:46,155] {taskinstance.py:1408} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-03-18 04:46:46,160] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 04:46:46,191] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20100307T040000, start_date=20220318T044645, end_date=20220318T044646
[2022-03-18 04:46:50,463] {local_task_job.py:212} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-03-18 04:46:50,466] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 644. PIDs of all processes in the group: [644]
[2022-03-18 04:46:50,467] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 644
[2022-03-18 04:46:50,469] {taskinstance.py:1408} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-03-18 04:46:50,475] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 04:46:50,507] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20100307T040000, start_date=20220318T044645, end_date=20220318T044650
[2022-03-18 04:46:50,565] {process_utils.py:70} INFO - Process psutil.Process(pid=644, status='terminated', exitcode=0, started='04:46:44') (644) terminated with exit code 0
